src/Stratis.Bitcoin.Features.Consensus/PowConsensusValidator.cs

```diff
@@ -564,8 +545,8 @@ public virtual void CheckBlock(ContextInformation context)
 
             Block block = context.BlockValidationContext.Block;
 
-            bool mutated = false;
-            uint256 hashMerkleRoot2 = this.BlockMerkleRoot(block, ref mutated);
+            bool mutated;
+            uint256 hashMerkleRoot2 = this.BlockMerkleRoot(block, out mutated);
```

Comment:
you could do the same here
`uint256 hashMerkleRoot2 = this.BlockMerkleRoot(block, bool out mutated);`